utils/htmlToPDF.ts

```diff
@@ -0,0 +1,23 @@
+import { create, CreateOptions } from "html-pdf";
+import { readFileSync } from "fs";
+
+/**
+ * converts hard coded html into a PDF and downloads the PDF locally
+ * @param {string} downloadPath download path of the generated pdf
+ */
+const htmlToPDF = (downloadPath: string): void => {
+    // path of the hard coded html or the html as a string
+    const htmlPath = "./test.html";
```

Comment:
Hard coded string will be provided

Reply:
If the hardcoded string will be provided then we should be taking in that string here